vcpkg/users/versioning.md

```diff
@@ -172,7 +172,7 @@ For an override to take effect, the overridden package must form part of the dep
   "builtin-baseline":"3426db05b996481ca31e95fff3734cf23e0f51bc",
   "overrides": [
     { "name": "fmt", "version": "6.0.0" },
-    { "name": "openssl", "version-string": "1.1.1h#3" }

```

Comment:
Note to self, find more instances of this in the documentation and change them.

Reply:
I grep'd for 'overrides' :)